### PR TITLE
Fix Few display issues

### PR DESCRIPTION
**Pull request by fwpushan**
1. Fix Asl language display issue for booking
2. Fix interpreter form layout issue.

### CODE REVIEW

**Review by will-freshworks — COMMENTED** [review]
We have some language fixing code in `frontend/src/constants/languages.ts` that might be worth consolidating. I worry that we haven't really solved the language display problem in a general way, given that we have two different fixes for different places now.

File: api/src/common/constant.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const reverseMap = (map: Map<string, string>) => {
+  const temp = new Map<string, string>();
+  for (const [key, value] of map) {
+    temp.set(value, key);
+  }
+  return temp;
+};
+
+export const ReverseLanguageMap: Map<string, string> = reverseMap(LanguageMap);
```
Comment (will-freshworks) [inline api/src/common/constant.ts:14]: I think a descriptively named function would be a better export, like `fixLanguageName`. `ReverseLanguageMap.get` isn't very clear what it's doing.

**Review by will-freshworks — CHANGES_REQUESTED** [review]

File: api/src/common/constant.ts (hunk 1 above)
Comment (fwpushan) [reply api/src/common/constant.ts]: Remove this variable fix this in front end but also rename the other one.

**Comment by fwpushan** [comment]
@will-freshworks Please review latest code, implement frontend only fix.

**Review by jerryfreshworks — APPROVED** [review]
looks good from me

**Pull request merged**